src/azure-cli/azure/cli/command_modules/cdn/custom.py

```diff
@@ -71,6 +91,315 @@ def update_endpoint(instance,
     return params
 
 
+# pylint: disable=too-many-return-statements
+def create_condition(match_variable=None, operator=None, match_values=None,
+                     selector=None, negate_condition=None, transform=None):
+    transforms = []
+    if transform is not None:
+        transforms = [transform]
```

Comment:
does your transform support list here? can we use append() to add element for a list?

Reply:
This could be a little confusing but we only support one transform at the moment. The list transforms should only contain one element.